src/app/components/home/home.component.html

```diff
@@ -3,8 +3,35 @@
   
     <div class="hero-body">
       <div class="container has-text-left">
-        <p class="title">Wooden Window is the simplest tool used for job searches</p>
-        <p class="subtitle">Create a profile and search pages of unless job openings nearby, or maybe even far away... It's up to you!</p>
+        <div class="columns">
+          <div class="column is-7">
+            <p class="title">Wooden Window is the simplest tool used for job searches</p>
+            <p class="subtitle">Create a profile and search pages of unless job openings nearby, or maybe even far away... It's up to you!</p>
+          </div>
+          <div class="column is-5">
+            <article class="message">
+              <div class="message-header">
+                <p class="has-text-centered">Get started now!</p>
+              </div>
+              <div class="message-body">
+                <h2 class="has-text-centered">We got the jobs, all you have to do is apply.</h2>
+                <div class="field">
+                  <div class="control">
+                    <p>Search Jobs</p>
+                    <input class="input is-primary" type="text" placeholder="Enter Job title or Keyword">
+                  </div>
+                </div>
+                <div class="field">
+                  <div class="control">
+                    <p>Search by location</p>
+                    <input class="input is-primary" type="text" placeholder="Enter Location">
+                  </div>
+                </div>
+                <button class="button is-primary">Submit</button>
```

Comment:
I had to look it up (was just interested how this would work going forward) and post Angular 4 you can use [routerLink] with a button so this will work once we get that component up and running.